jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -305,7 +327,8 @@ boolean isRPCServer() {
               startTimeMicroseconds,
               startTimeNanoTicks,
               computeDurationViaNanoTicks,
-              tags);
+              tags,
+              references);
```

Comment:
Does the `parent_id` offer that much of a benefit, to warrant this processing in the client? Seems like it would be easier just to use the references.

If the `parent_id` is required for the UI currently, then couldn't this logic be implemented in the server, so that once the UI no longer requires it, it is simply a case of changing the logic in one place?